.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+defaults:
+    run:
+        # To load bashrc
+        shell: bash -ieo pipefail {0}
+
+on: [push, pull_request]
```

Comment:
```yml
on:
  push:
    branches:
      - main
  pull_request:
```

Is probably the better setting ? Currently it will run every workflow twice on pull request branches... once for the push and once for the update of the pull request triggered by the push.... With the settings above it would only run once for pull request pushes/updates and once when it's actually merged....

Same for the other two workflows I guess

Reply:
I got it, I'll implement it